client/a8c-for-agencies/components/a4a-feedback/provide-feedback/style.scss

```diff
@@ -0,0 +1,56 @@
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+.a4a-provide-feedback__modal {
+	.a4a-modal__main {
```

Comment:
To reduce nesting, we can merge this selector instead.

`.a4a-provide-feedback__modal .a4a-modal__main`